runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -297,6 +306,41 @@ private void playerSkillLookup(ChatMessageType type, SetMessage setMessage, Stri
 		}
 	}
 
+	/**
+	 * Calculates a simple equation and changes the original message to the
+	 * response.
+	 *
+	 * @param messageNode The chat message containing the command.
+	 * @param eqn         The equation to calculate.
+	 */
+	private void calculate(MessageNode messageNode, String eqn)
+	{
+		String result;
```

Comment:
Why are you declaring result outside of try catch when it is not used outside?

Reply:
Was doing it for debugging purposes. Changed.